bin/generateTestHeapHeaders.js

```diff
@@ -68,7 +68,7 @@ if (argv.type === 'sync') {
     dataBody = (JSON.stringify(drainData));
 }
 
-const hmac = CryptoJS.enc.Base64.stringify(
+const hmac = CryptoJS.enc.Hex.stringify(
```

Comment:
Im not quite sure about about this. I think we just want to get rid of the encoding instead of changing it? So theoretically just match whatever is genenerated from `CryptoJs.HmacSHA256(....`

Reply:
@colbysadams do you remember the original intent of this ticket lol

Reply:
https://replit.com/@yilmazhuseyin/cryptojstest#index.ts